src/contracts/GPv2Settlement.sol

```diff
@@ -372,6 +372,19 @@ contract GPv2Settlement {
         }
     }
 
+    /// @dev Claims order gas refunds by freeing storage for all encoded order
+    /// gas refunds.
+    function claimOrderRefunds(bytes calldata encodedOrderRefunds) internal {
+        uint256 orderUidLength = GPv2Encoding.ORDER_UID_LENGTH;
```

Comment:
When I checked the assembly in Remix, it looked like it was doing a `JMP` for each iteration to push the value onto the stack instead of just doing a `DUP`.

However, I realize now that it was because I forgot to turn optimizations on :see_no_evil:. This shouldn't be needed so I'll use the constant directly (it actually turns it into a `PUSH1 0x38` at all locations which is a really nice 2 gas and definitely cheaper).

Reply:
Fixed!